### PR TITLE
feat: Wrap addons form in FormTemplate

### PR DESCRIPTION
**Pull request by chriswk**
This PR updates the addon form to use the FormTemplate component to make the addon form align with the rest of our forms.

### CODE REVIEW

**Review by sighphyre — APPROVED** [review]
LGTM

File: src/component/addons/AddonForm/AddonForm.styles.tsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+});
+
+export const StyledButtonContainer = styled('div')({
+    marginTop: 'auto',
+    display: 'flex',
+    justifyContent: 'flex-end',
+});
+
+export const StyledButtonSection = styled('section')(({ theme }) => ({
+    padding: '10px 36px',
```
Comment (NicolaeUnleash) [inline src/component/addons/AddonForm/AddonForm.styles.tsx:35]: i missed this one -> if you can change this to padding-top: 16px (or 1rem)

File: src/component/addons/AddonForm/AddonForm.tsx [hunk 2]
```diff
@@ -222,97 +241,113 @@ export const AddonForm: VFC<IAddonFormProps> = ({
...
+                        />
+                    </StyledFormSection>
+                    <StyledFormSection>
+                        <StyledHelpText>
+                            What is your addon description?
+                        </StyledHelpText>
+
+                        <StyledTextField
+                            size="small"
+                            style={{ width: '80%' }}
```
Comment (NicolaeUnleash) [inline src/component/addons/AddonForm/AddonForm.tsx:280]: and this one to be 100%

**Comment by sjaanus** [comment]
LGTM!

File: src/component/addons/AddonForm/AddonForm.styles.tsx (hunk 1 above)
Comment (chriswk) [reply src/component/addons/AddonForm/AddonForm.styles.tsx:35]: will do

**Pull request merged**